config/helm/chart/default/templates/Common/extensions-controller/serviceaccount-extensions-controller.yaml

```diff
@@ -0,0 +1,24 @@
+{{- include "dynatrace-operator.platformRequired" . }}
+{{ if eq (include "dynatrace-operator.partial" .) "false" }}
+# Copyright 2021 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dynatrace-extensions-controller
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "dynatrace-operator.extensionsControllerLabels" . | nindent 4 }}
+{{ end }}
```

Comment:
funny thing, I know helm supports both, but in second file we use `{{- end -}}`, so is it possible to sync it?

Reply:
I see on the templates we mostly use `{{ end }}`, so IMO this fits more the standard.
[I had to google about it](https://stackoverflow.com/questions/67293728/what-is-the-difference-between-helm-syntax-something-and-something), adding the `-` puts an empty space if rendered, so I would say in this case makes sense to have only {{ end }}.

Reply:
sure, my point is to have the same syntax in these new files. Maybe later, you can create a new PR to remove from all files.

Reply:
:+1::+1::+1:
As it is now the PR, it follows the standards we already had in this files. We can later create a followup PR and update them if we want to normalize them (but IMO is fine as it is).